carina-api/src/main/java/com/qaprosoft/carina/core/foundation/api/AbstractApiMethodV2.java

```diff
@@ -142,7 +139,7 @@ public Response callAPIExpectSuccess() {
 
     /**
      * Sets path to .properties file which stores properties list for declared API method
-     * 
```

Comment:
please use proper carina formatter in all your updates to avoid changes like this.
https://github.com/qaprosoft/carina/blob/master/carina_formatter.xml

Reply:
please remove changes like this as I already asked